This patch fixes .so links in man pages.
It's needed because of the configure option -

    --program-suffix=${_php_version}

The changes made by this patch should be the same as the changes made by this
block of code -

    _regexp='^[[:space:]]*\.so[[:space:]]'
    for file in $(grep -l -e "$_regexp" -R .); do
    	vsed -i "$file" -e "/$_regexp/"'s=^[[:space:]]*\.[^.]*=&'${_php_version}=
    done

Where _php_version is defined in the template file.

--- a/ext/phar/phar.phar.1.in
+++ b/ext/phar/phar.phar.1.in
@@ -1 +1 @@
-.so man1/phar.1
+.so man1/phar8.3.1
--- a/sapi/cgi/php-cgi.1.in
+++ b/sapi/cgi/php-cgi.1.in
@@ -1 +1 @@
-.so man1/php.1
+.so man1/php8.3.1
